x-pack/plugins/alerting/server/monitoring/rule_last_run_service.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { PublicLastRunSetters } from '../types';
+
+export interface RuleLastRunResults {
+  errors: string[];
+  warnings: string[];
+  outcomeMessage: string;
+}
+
+export class RuleLastRunService {
```

Comment:
I am not really in love with the name, I know it is lame. What do you think about `RuleExceptionRunService`? or maybe you have a better idea

Reply:
This service do exactly what we talk about!!!

Reply:
Renamed it to `RuleExecutionService`. I think it's clearer. What do you think?

Reply:
I talked with the team, we all like `RuleResultService`. we will be happy clams